R/nicks.R

```diff
@@ -0,0 +1,93 @@
+
+nicks <-
```

Comment:
Should nicks be a function returning the nicks to not have to declare a global variable?

Reply:
does not matter much in this case

Reply:
although to test the on-demand nick update, a function is better bc it can be mocked easier

Reply:
I meant a function returning this named vector, so apparently it would have no advantage :-)